### PR TITLE
chore: deprecate disk_space option

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Should be removed a long ago.

### CODE REVIEW

File: internal/schemautil/service.go [hunk 1]
```diff
@@ -107,9 +107,10 @@ func ServiceCommonSchema() map[string]*schema.Schema {
 		"disk_space": {
 			Type:          schema.TypeString,
 			Optional:      true,
-			Description:   "Service disk space. Possible values depend on the service type, the cloud provider and the project. Therefore, reducing will result in the service rebalancing.",
+			Description:   "Additional disk space. Possible values depend on the service type, the cloud provider and the project. Therefore, reducing will result in the service rebalancing.",
```
Comment (mhoffm-aiven) [inline internal/schemautil/service.go:110]: Wait isnt it total disk space?

Comment (byashimov) [reply internal/schemautil/service.go]: Thanks.

**Comment by Serpentiel** [comment]
please fix the [syntax](https://github.com/aiven/terraform-provider-aiven/actions/runs/4896027468/jobs/8742321515?pr=1151#step:4:94) error

File: generator/main.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package main
```
Comment (Serpentiel) [inline generator/main.go:1]: looks like these files shouldn't be there 🤔

**Review by Serpentiel — CHANGES_REQUESTED** [review]

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**